nsv13/code/modules/overmap/fighters/tug.dm

```diff
@@ -87,9 +81,36 @@
 /obj/vehicle/sealed/car/realistic/fighter_tug/proc/can_launch_fighters()
 	return TRUE
 
+//This proc is cringe. Raw, unadulterated cringe. It'll do for now
```

Comment:
so what was wrong with the angle2dir proc in code/__HELPERS/type2type.dm ?

Reply:
youre kidding me, right?